src/Utils/Utils.php

```diff
@@ -266,37 +266,30 @@ public static function ord(string $char, string $encoding = 'UTF-8'): int
             $char = mb_convert_encoding($char, 'UCS-4BE', $encoding);
         }
 
+        // @phpstan-ignore-next-line format string is statically known to be correct
```

Comment:
Overall it's better to use `assert()` instead of overriding types using `@var` or ignoring the whole line.

`assert` has no performance impact and allows to pinpoint the exact issue.